.github/workflows/release.yml

```diff
@@ -60,11 +59,7 @@ jobs:
       - name: Setup Bazel
         # Ubuntu bazel is run inside of the docker image
         if: matrix.os != 'ubuntu-18.04'
-        # MacOS VMs share IP addr on GH actions. This bazel installation authenticates token to prevent api rate limiting.
```

Comment:
So this is my only concern. The specific bazel setup here did a token authentication to prevent rate limiting of bazel installations on MacOS VMs. This is such a nice clean-up I think it's okay to go forward with this but if we see builds breaking due to github limits we'll need to roll this back.

Reply:
Oh I see. I originally think it's github-action-specific issue. Thanks for reminding.